04_glm_query/server/glm_query/config.py

```diff
@@ -14,8 +14,8 @@
         "geth_address": "https://bor.golem.network",
         "glm_contract_address": "0x0B220b82F3eA3B7F6d9A1D8ab58930C064A2b5Bf",
     },
-    "rinkeby": {
-        "name": "Rinkeby",
+    "goerli": {
+        "name": "Goreli",
```

Comment:
@shadeofblue I didn't dig out the geth_address and the contract address for Goerli. Would you be able to update that before merge?